_includes/header.html

```diff
@@ -25,24 +25,43 @@
           </svg>
           <span class="mk-main-header__toggle-text">menu</span>
         </button>
+      </nav>
```

Comment:
Any reason for closing navigation earlier ?

Reply:
While trying to center the nav links on mobile responsiveness, I couldn't get them to center perfectly so I decided to separate the toggle button and create a separate div (ul) making the nav links stand independently.
Thanks for raising this, I would change the nav to div for best practice